solutions/usaco-971.mdx

```diff
@@ -0,0 +1,245 @@
+---
+id: usaco-971
+source: USACO Gold 2019 December
+title: Moortal Cowmbat
+author: Sofia Yang
+---
+
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_cowmbat_gold_dec19.html)
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(M^2 + N M^2)$
+
+<LanguageSection>
+
+<CPPSection>
+
+```cpp
+CodeSnip{CPP Short Template}
```

Comment:
...
i'm not mad, just disapointed

Reply:
I thought I was supposed to use the codesnippet?
I can remove it if you want, though

Reply:
Alright, the codesnippet is gone, now you can finally rest in peace 🤣